it/spring/boot-tomcat8.5/build.gradle

```diff
@@ -17,21 +6,43 @@ buildscript {
         classpath 'org.springframework.boot:spring-boot-gradle-plugin:2.0.2.RELEASE'
     }
 }
+
 apply plugin: 'java'
 apply plugin: 'org.springframework.boot'
 
-sourceSets {
-    main.java.srcDir project(':it:spring:boot-tomcat').sourceSets.main.java
-    main.resources.srcDir project(':it:spring:boot-tomcat').sourceSets.main.resources
-    test.java.srcDir project(':it:spring:boot-tomcat').sourceSets.test.java
+dependencyManagement {
+    // Override Tomcat versions.
+    dependencies {
+        dependencySet(group: 'org.apache.tomcat.embed', version: '8.5.31') {
+            entry 'tomcat-embed-core'
+            entry 'tomcat-embed-jasper'
+            entry 'tomcat-embed-el'
+        }
+    }
 }
 
 dependencies {
     compile project(':spring:boot-starter')
     compile project(':tomcat')
-    compile project(':thrift0.9')
     compile('org.springframework.boot:spring-boot-starter-web') {
         exclude group: 'org.springframework.boot', module: 'spring-boot-starter-tomcat'
     }
     testCompile('org.springframework.boot:spring-boot-starter-test')
 }
+
+// Use the sources from ':it:spring:boot-tomcat'.
+// NB: We should not add these in the 'sourceSets' directive because that will make the directories mentioned
+//     below are added to two projects and having a source directory with two output directories will confuse
+//     the IDE.
```

Comment:
`IntelliJ IDEA version x.`

Reply:
I think it was around IntelliJ IDEA 12, but I forgot. Let me update anyway.

Reply:
Actually, it's not 12 but some other version before they used year.

Reply:
Well, I'm not really sure. Let me just mention without version numbers.

Reply:
Fixed